File: niro/src/main/java/nl/uva/se/sc/niro/gui/util/ErrorUtil.scala
package nl.uva.se.sc.niro.gui.util

import nl.uva.se.sc.niro.parser.ParseErrorInfo

import scala.collection.mutable.ListBuffer

object ErrorUtil {
  def toString(parseErrors: ListBuffer[ParseErrorInfo]): String = {
    val message = new StringBuilder
    for (errorInfo <- parseErrors) {
      message.append(errorInfo.toString)
      message.append("\n")
    }
    message.toString
  }
}

File: niro/src/main/java/nl/uva/se/sc/niro/gui/util/ErrorUtil.java
package nl.uva.se.sc.niro.gui.util;

import nl.uva.se.sc.niro.parser.ParseErrorInfo;

import java.util.List;

public class ErrorUtil {
    public static String toString(List<ParseErrorInfo> parseErrors) {
        StringBuilder message = new StringBuilder();
        for (ParseErrorInfo errorInfo : parseErrors) {
            message.append(errorInfo.toString());
            message.append("\n");
        }
        return message.toString();
    }
}
